src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import Calculator from '../components/Calculator.js';
```

Comment:
- [ ] Please note that the imported file name isn't matching the existing one which is representing the following error at the image down below, please let's fix it 😄 🙏 
![Screenshot](https://user-images.githubusercontent.com/47625407/129813226-238d99db-f668-444c-b12d-22e2b743ae21.png)

Reply:
Fixed